CEXT-4720: Update Checkout Starter Kit configuration steps (#396)
AdobeDocs/commerce-extensibility: src/pages/starter-kit/checkout/connect.md

Depending on your Adobe Commerce setup, there are two options to authenticate and communicate with App Builder:

- [Adobe Identity Management Service (IMS)](#adobe-identity-management-service-ims)

- &#8203;<Edition name="paas" /> [Commerce Integration](#create-a-commerce-integration)

> If IMS authentication is detected, it has precedence over Commerce integration. However, if neither option is detected or configured, then client instantiation will fail.

Reviewer: ```suggestion
If IMS authentication is detected, it has precedence over Commerce integration authentication. However, if neither option is detected or configured, then client instantiation will fail.
```

1. Access your IMS credentials through the [Adobe Developer Console](https://developer.adobe.com/console). Select the project and workspace you set up during the [initial configuration](./getting-started.md#initial-configuration). Then click **OAuth Server-to-Server** in the side-navigation menu.

>    Copy the IMS credentials to the `.env` file in the root of the project.

Reviewer: ```suggestion
1. Copy the IMS credentials to the `.env` file in the root of the project.
```

>    **NOTE**: These credentials are automatically filled out in [Configure OAuth Server-to-Server Credential](./configure.md#configure-oauth-server-to-server-credential).

Reviewer: ```suggestion
   **NOTE**: These credentials are automatically populated in [Configure OAuth Server-to-Server Credential](./configure.md#configure-oauth-server-to-server-credential).
```

   OAUTH_IMS_ORG_ID=<img org>
   ```

1. Add a technical account with server-to-server credentials to the Commerce Admin with the appropriate permissions using the [Admin User Creation Guide](https://experienceleague.adobe.com/en/docs/commerce-admin/systems/user-accounts/permissions-users-all#create-a-user).

> 1. When associating the user with the account, find your `Technical Account email` as a part of generated IMS credentials with following pattern: `<technical-account>@techacct.adobe.com` and use that value in the **Email** field during user creation:

Reviewer: ```suggestion
1. When associating the user with the account, find your **Technical Account email** as a part of generated IMS credentials with following pattern: `<technical-account>@techacct.adobe.com` and use that value in the **Email** field during user creation:
```
